framework/auth/campaigns.py

```diff
@@ -9,7 +9,7 @@
 CAMPAIGNS = ImmutableDict({
     'prereg': {
         'system_tag': 'prereg_challenge_campaign',
-        'redirect_url': lambda: web_url_for('prereg_landing_page'),
+        'redirect_url': lambda: web_url_for('prereg_landing_page', _absolute=True),
```

Comment:
There's a slim change this will break ERPC. Even if not, this will need to be updated for ERPC when that goes out (this release cycle).